dts/bindings/ieee802154/ti,cc13xx-cc26xx-ieee802154-subghz.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI SimpleLink CC13xx / CC26xx IEEE 802.15.4 node (sub-GHz)
+
+compatible: "ti,cc13xx-cc26xx-ieee802154-subghz"
```

Comment:
@cfriedt is subghz really just a mode?  Seems like its not a different device?  Can you really have both subghz and non-subghz drivers enabled at the same time?